tools/nni_cmd/config_schema.py

```diff
@@ -142,6 +142,16 @@ def setPathCheck(key):
         Optional('includeIntermediateResults'): setType('includeIntermediateResults', bool), 
         Optional('gpuNum'): setNumberRange('gpuNum', int, 0, 99999),
     },
+    'PPOTuner': {
+        'builtinTunerName': 'PPOTuner',
+        'classArgs': {
+            'optimize_mode': setChoice('optimize_mode', 'maximize', 'minimize'),
```

Comment:
I think there the hyper-parameter in PPO may affect the performance of this tuner. such as "learning rate"...